source/content/guides/filesystem/03-large-files.md

```diff
@@ -37,7 +42,7 @@ A code repository larger than 2GB increases the possibility of Git errors when c
 
 ## Large Files
 
-The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure is not optimized to store and deliver large files.
+The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure are not optimized to store and deliver large files.
```

Comment:
```suggestion
The [Pantheon Filesystem](/guides/filesystem) and file serving infrastructure are not optimized to store and deliver large files.  The maximum size for any single file is 256 MiB.
```
Documenting the hard limit here may be redundant to the table below, but IMO it's better to call this out clearly (& repeatedly? :P ).